apiserver/facades/client/applicationoffers/state.go

```diff
@@ -62,7 +62,6 @@ type Backend interface {
 	OfferConnections(string) ([]OfferConnection, error)
 	SpaceByName(string) (Space, error)
 	User(names.UserTag) (User, error)
-	UserPermission(subject names.UserTag, target names.Tag) (permission.Access, error)
```

Comment:
drive by - found with unconvert

Reply:
Once https://github.com/juju/juju/pull/12000 has landed, you can rebase this away.

Reply:
done